### ISSUE TITLE
Windows client 4.1.2 does not work with OPNsense server

### ISSUE THREAD

**Issue by Khoramyar** [shadowsocks/shadowsocks-windows#2037]
- Shadowsocks is an open source project for noneprofit. If you bought the service from a provider, please contact them. / 影梭（Shadowsocks）是一个开源非盈利项目，不提供任何托管服务。如果你是从服务提供商购买的服务，请联系他们
- If you have questions rather than Shadowsocks Windows client, please go to / 如果你有非影梭Windows客户端相关的问题，请去 https://github.com/shadowsocks
- Please read Wiki carefully, especially / 提问前请先阅读wiki https://github.com/shadowsocks/shadowsocks-windows/wiki/Troubleshooting.
- Or search from Issue Board / 或在Issue Board中搜索 https://github.com/shadowsocks/shadowsocks-windows/issues?utf8=%E2%9C%93&q=is%3Aissue
- Please answer the following questions before submission. Questions lack of details will be closed. / 请按照以下格式描述你的问题，描述不清的问题将会被关闭

### Shadowsocks version / 影梭版本
OPNsense version 18.7.4 (latest) and Shadowsocks plugin version 1.0 (latest)

### Environment(Operating system, .NET Framework, etc) / 使用环境（操作系统，.NET Framework等）
Windows 10 (latest stable release)

### Steps you have tried / 操作步骤
The server works fine with all clients on other platforms, and also windows clients untill 4.1.1
When I test 4.1.2 traffic does not pass and I maintain original network IP. 

### What did you expect to see? / 期望的结果
I expected the client to work normally

### What did you see instead? / 实际结果
Shadowsocks tray icon shows it's connected but it is not.

### Config and error log in detail (with all sensitive info masked) / 配置文件和日志文件（请隐去敏感信息）

**Comment by chenshaoju** [comment]
4.1.1 and 4.1.2 do not have much changed, The protocol is the same.
https://github.com/shadowsocks/shadowsocks-windows/compare/4.1.2...master

There are any logs? Please paste here, And hide your server information.

**Comment by Khoramyar** [comment]
Sorry, my bad. The problem was something else, not the update. 

Please close this issue. Thank you.

**Comment by studentmain** [comment]
Use global proxy mode (when it activated, tray icon is blue, not white) and retry. PAC mode will only proxy websites in a list.

BTW, shadowsocks is a stateless protocol, 'connected' just means ss-windows client opened local proxy, modify system proxy config, and ready to send encrypted packet to server.

**Comment by Khoramyar** [comment]
> Use global proxy mode (when it activated, tray icon is blue, not white) and retry. PAC mode will only proxy websites in a list.
> 
> BTW, shadowsocks is a stateless protocol, 'connected' just means ss-windows client opened local proxy, modify system proxy config, and ready to send encrypted packet to server.

Yes I am aware of the settings and what they do. The issue was more basic that this. I rushed for opening an issue here, I had to spend a little more time troubleshooting. :)
